language/movie_nl/README.md

```diff
@@ -0,0 +1,151 @@
+come up with the most and least popular actors/directors.
+
+### Movie document
+We define a movie document as a set of reviews. These reviews are individual
+sentences and we use the NL API to extract the sentences from the document. See
+an example movie document below.
+
+```
+  Sample review sentence 1. Sample review sentence 2. Sample review sentence 3.
+```
+
+### Sentences and Sentiment
+Each sentence from the above document is assigned a sentiment as below.
+
+```
+  Sample review sentence 1 => Sentiment 1
+  Sample review sentence 2 => Sentiment 2
+  Sample review sentence 3 => Sentiment 3
+```
+
+### Sentiment computation
+The final sentiment is computed by simply adding the sentence sentiments.
+
+```
+  Total Sentiment = Sentiment 1 + Sentiment 2 + Sentiment 3
+```
+
+
+### Entity extraction and Sentiment assignment
+Entities with type `PERSON` are extracted from the movie document using the NL
+API. Since these entities are mentioned in their respective movie document,
+they are associated with the document sentiment.
+
+```
+  Document 1 => Sentiment 1
+
+  Person 1
+  Person 2
+  Person 3
+
+  Document 2 => Sentiment 2
+
+  Person 2
+  Person 4
+  Person 5
+```
+
+Based on the above data we can calculate the sentiment associated with Person 2:
+
+```
+  Person 2 => (Sentiment 1 + Sentiment 2)
+```
+
+## Movie Data Set
+We have used the Cornell Movie Review data as our input. Please follow the instructions below to download and extract the data.
+
+### Download Instructions
+
+```
+ $ curl -O http://www.cs.cornell.edu/people/pabo/movie-review-data/mix20_rand700_tokens.zip
+ $ unzip mix20_rand700_tokens.zip
+```
+## Command Line Usage
+In order to use the movie analyzer, follow the instructions below. (Note that the `--sample` parameter below runs the script on
+fewer documents, and can be omitted to run it on the entire corpus)
+
+### Install Dependencies
+
+Install [pip](https://pip.pypa.io/en/stable/installing) if not already installed.
+
+Then, install dependencies by running the following pip command:
+
+```
+$ pip install -r requirements.txt
+```
+### How to Run
+
+```
+$ python main.py --inp "tokens/*/*"
+                 --sout sentiment.json
+                 --eout entity.json
+                 --sample 5
+```
+
+You should see the log file `movie.log` created.
+
+## Output Data
+The program produces sentiment and entity output in json format. For example:
+
+### Sentiment Output
+```
+  {
+    "doc_id": "cv310_tok-16557.txt",
+    "sentiment": 3.099,
+    "label": -1
+  }
+```
+
+### Entity Output
+
+```
+  {
+    "name": "Sean Patrick Flanery",
+    "wiki_url": "http://en.wikipedia.org/wiki/Sean_Patrick_Flanery",
+    "sentiment": 3.099
+  }
+```
+
+### Entity Output Sorting
+In order to sort and rank the entities generated, use the same `main.py` script. For example,
+this will print the top 5 actors with negative sentiment:
+
+```
+$ python main.py --inp entity.json
```

Comment:
first of all, better to put \ at end of each line to make easier copy paste.

secondly I get

TypeError: coercing to Unicode: need string or buffer, NoneType found